src/components/Signin.jsx

```diff
@@ -0,0 +1,276 @@
+                                    </button>
+                                )}
+                                {status === "ERROR" && (
+                                    <h5 className="text-danger">Oops! There was an error.</h5>
+                                )}
+                            </div> */}
+                            <div className="form-group text-center">
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignIn
+                              </Button>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-danger rounded-pill shadow"
+                                width="60%"
+                                marginLeft="5px"
+                              >
+                                Forgot your password?
+                              </Button>
+                            </div>
+                            <div className="form-group text-center">
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <AuthLinkWrapper >
+                                <h2><span>OR</span></h2>
+                              </AuthLinkWrapper>
+                              <DisplayForLinks >
+                                  <li>
+                                    <AuthLink  
+                                      color ="white" bgColor="#3B5998" 
+                                      href="#" hoverColor="white"
+                                      hoverBgColor="#203797" className="btn"
+                                    >
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      bgColor="#2D2C2C" color="white" 
+                                      hoverBgColor="#4A4A4C" href="#" 
+                                      hoverColor="white"    className=" btn"
+                                    >
+                                      <i className="fa fa-github "></i> Signin with Github
```

Comment:
For all the options. 
Make AuthLink a flex box. - display: flex, justify-content space around
icon - flex: 1 20px or 30px.
text - flex - 1 505 or 60%;
All the options text and icons should be aligned vertically.

Reply:
vertically ? , it wont look good .